tests/test_documentation_build.py

```diff
@@ -0,0 +1,24 @@
+"""Tests for documentation build."""
```

Comment:
@Zeroto521 and @samukweku this is the automated hack I have in place to check that our docs are built correctly automatically... at least for the general functions page. We may want to do a more comprehensive thing later.